blackjax/util.py

```diff
@@ -171,6 +171,8 @@ def run_inference_algorithm(
         A transformation of the trace of states to be returned. This is useful for
         computing determinstic variables, or returning a subset of the states.
         By default, the states are returned as is.
+    streaming
+        if True, `run_inference_algorithm` will take a streaming average of the value of transform, and return that average instead of the full set of samples. This is useful when memory is a bottleneck.
```

Comment:
also no diagnostic is return when set to True

Reply:
This is true. Fixing this was slightly tricky, and I think should be deferred to an upcoming change which will change transform to also take info. I didn't want to do this here because it breaks other parts of the codebase.